ios/Runner/GoogleService-Info.plist

```diff
@@ -3,17 +3,19 @@
 <plist version="1.0">
 <dict>
 	<key>CLIENT_ID</key>
-	<string>366914936426-daqh0o5pqqqb58tt71b3ms1lm0p82hge.apps.googleusercontent.com</string>
+	<string>366914936426-eop45t8oepqu48v4fsp8frgqo0h0fmqd.apps.googleusercontent.com</string>
 	<key>REVERSED_CLIENT_ID</key>
-	<string>com.googleusercontent.apps.366914936426-daqh0o5pqqqb58tt71b3ms1lm0p82hge</string>
+	<string>com.googleusercontent.apps.366914936426-eop45t8oepqu48v4fsp8frgqo0h0fmqd</string>
+	<key>ANDROID_CLIENT_ID</key>
+	<string>366914936426-k4i8n24qb9eu1ug0cqs0rcfvc3pq1fh3.apps.googleusercontent.com</string>
 	<key>API_KEY</key>
 	<string>AIzaSyCl7WOc4uKnUqlvCilb9Zt9IkBU9ib_Tn8</string>
 	<key>GCM_SENDER_ID</key>
 	<string>366914936426</string>
 	<key>PLIST_VERSION</key>
 	<string>1</string>
 	<key>BUNDLE_ID</key>
-	<string>com.joinseeds.seedswallet</string>
+	<string>com.joinseeds.seedslight</string>
```

Comment:
Do we have that app in firebase with dynamic links configured?

Reply:
I need to re test this but I did create a new firebase app